pkg/process/procutil/data_scrubber_windows.go

```diff
@@ -0,0 +1,85 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+package procutil
+
+import (
+	"strings"
+)
+
+// Extensions in which windows defaults the exec commands. Additional extensions can be included. 
+var (
+	winDotExec= []string{
+		".com",".exe",".bat",".cmd",".vbs", ".vbe",".js",".jse",".wsf",".wsh",".psc1", ".ps1"} 
+)
+
+// stripWindowsArgs removes the arguments of the commands if they identify any of the file extensions that windows defaults to.
+// If the command doesn't identify the extension and cannot split the exec command from the args, it will return the existing characters until the first empty space occurrence.
+
+func (ds *DataScrubber) stripArguments(cmdline []string)[]string {
+	
+	argLength := len(cmdline)
+	
+		strCmdline := (cmdline[0] + " ")
+	
+	// case 1: Uses extensionParser() to format and search the command. 
+		if argLength == 1 && !strings.HasPrefix(strCmdline, "\"") {
+			strippedCmdline := extensionParser(strCmdline, winDotExec)
+			cmdline = cleanUp(strippedCmdline)
+			return cmdline
+		
+		}
+	// case 2: Uses quotesFinder() to search for any existing pair of double quotes ("") existing in the string as characters and return the content between them. 
+		if argLength == 1 && strings.HasPrefix(strCmdline, "\""){
+			strippedCmdline := quotesFinder(strCmdline)
+			cmdline = cleanUp(strippedCmdline)
+			return cmdline
+	
+		}
+	// case 3: Process a cmdline with multiple strings and use extensionParser() to format and search the command. 
+		if 	argLength > 1 && !strings.HasPrefix(strCmdline, "\""){
+			strippedCmdline := extensionParser(strCmdline, winDotExec)
+			cmdline = cleanUp(strippedCmdline)
+			return cmdline
+					
+		}
+	
+		return cmdline
+	}	
+	
+	// remove any extra space at the end cmdline before been returned to Scrubber 
+	func cleanUp(strippedCmdline string)[]string{
+		cmdline := []string{strings.TrimSuffix(strippedCmdline," ")}
+		return cmdline
+	}
```

Comment:
This function doesn't really do or add much value. You should just inline it.

Reply:
I'm adding this function since I'm using after each case, so to don't repeat the code three times (lines 27, 34 and 41) I prefer to set a function. 
Based on this, do you still believe will be better just inline?

Reply:
Is this clean up due to this? https://github.com/DataDog/datadog-agent/pull/22445/files#r1475068672

Reply:
Marking as resolved as there are no more cleanUp() instances in most recent commit.